src/eip4788_deployment/eip4788_deployment_launcher.star

```diff
@@ -0,0 +1,32 @@
+PYTHON_IMAGE = "python:3.11-alpine"
+EIP4788_DEPLOYMENT_SERVICE_NAME = "eip4788-contract-deployment"
+
+
+def deploy_eip4788_contract_in_background(plan, sender_key, receiver_key, el_uri):
```

Comment:
perhaps this can be pulled out in a `run_python_script_in_background` method that gets shared between this and `mev_custom_flood`; don't have to do it right now but maybe worth trackign a TODO

Reply:
just more co-reuse. i'd toss a TODO for now and leave it at that

Reply:
issue here: https://github.com/kurtosis-tech/ethereum-package/issues/287